knn-banking-product-recommender/deployment/demo.py

```diff
@@ -0,0 +1,94 @@
+from typing import Dict
+
+import gradio as gr
+import highwind
+
+# Replace this with the unique id of your UseCase on Highwind
+# This can be found by viewing the UseCase on the Highwind frontend and extracting it from the URL:
+# https://frontend.dev.highwind.cloud/use_cases/{{use-case-id-to-copy}}
```

Comment:
```suggestion
# https://frontend.zindi.highwind.cloud/use_cases/{{use-case-id-to-copy}}
```

Maybe also worth noting here that without overriding the Highwind SDK env vars, that this will default to the UAT/Zindi environment.